src/spec/Fastly/Adapter/Guzzle/GuzzleAdapterSpec.php

```diff
@@ -14,6 +14,6 @@ function it_is_initializable()
 
     function it_returns_a_psr7_response()
     {
-        $this->send('POST', 'http://endpoint.com/some/uri')->shouldHaveType('Psr\Http\Message\ResponseInterface');
+        $this->send('POST', 'http://smartframe.io')->shouldHaveType('Psr\Http\Message\ResponseInterface');
```

Comment:
We would prefer to not reference your own company in these files, but instead opt for something either generic (like `https://httpbin.org`) or keep it set to what it is currently.

Reply:
Hi, sure previous link did not work, I will change it to https://httpbin.org as you suggest.